zipp.py

```diff
@@ -61,6 +61,39 @@ def _difference(minuend, subtrahend):
     """
     return itertools.filterfalse(set(subtrahend).__contains__, minuend)
 
+class PickleableClass():
```

Comment:
I like what you've done here largely separating the concern of pickleability from the implementation details.